docs/release/dev/sp-default-browser.md

```diff
@@ -0,0 +1,9 @@
+## Support opening url in os default browser
+
+Now we can open url in external os default browser.
+
+```typescript
+browser.shellExecute("https://google.com");
```

Comment:
Not sure it's reasonable to add this to `browser` module. I corrected the task. See the original issue.